### PR TITLE
feat: update composer, remove dependecie hirak/prestissimo(not needed) and add instructions

### PR DESCRIPTION
**Pull request by brandovidal**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,4 @@
 ###> Other files and folders ###
 .idea
 ###< Other files and folders ###
+*.lock
```
Comment (juanwilde) [inline .gitignore]: This file should be kept as it prevents versioning issues for other developers cloning the repository, so if you clone it and then run `composer install` you should have the same versions I'm using in the YouTube tutorial

File: README.md [hunk 2]
```diff
@@ -1,5 +1,66 @@
...
+```
+
+Install packages
+
+```shell
+composer install
+```
+
+
+### Additional
```
Comment (juanwilde) [inline README.md]: Maybe "Optional" would be a better word as you can use also `docker` command to check stuff :)

Comment (brandovidal) [reply README.md]: Yes, you are have the reason, I've correct this part

File: .gitignore (hunk 1 above)
Comment (brandovidal) [reply .gitignore]: Alright, I've correct this part to avoid possible errors.

**Review by juanwilde — APPROVED** [review]

**Pull request merged**